database/factories/ResetPasswordFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\ResetPassword;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class ResetPasswordFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = ResetPassword::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
```

Comment:
When you add factory, you should fully fill in definitions else it will not work and it will confuse your teammates

Reply:
Or just remove factory if it's useless